### ISSUE TITLE
Radio input Sketch symbol is taller than the live component

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/vets-design-system-documentation#1294]
# Bug Report

- [ x ] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The Sketch symbol is 36 pixels tall but when I inspect the radio input on the website, it's 24px tall. It has an 8px top margin, so if the symbol includes the margin, it needs to be reduced by 2px, and the margin in the symbol needs to be swapped (currently it has a margin on the bottom)

This makes it hard to align spacing in mockups to the components without breaking the symbols.

## What I expected to happen

The Sketch symbol should mirror what code renders so we don't need to break the symbol and developers don't have to wonder if they need to add custom margins when they see a design

## Reproducing
- Formation version: current
- Device: MacBook
- Browser: Chrome

Steps to reproduce:

1. Insert the Form elements / radio / unselected symbol from the Sketch pattern library; note the height is 36px.
2. Use dev tools in your browser to [inspect the radio button pattern example](https://design.va.gov/components/form/radio-button) on design.va.gov and note the height of the `<va-radio-option>` element is 24px, with a top margin of 8px

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by GnatalieH** [comment]
Hi @andaleliz. On the DST we are in the process of consuming [USWDS radio buttons](https://designsystem.digital.gov/components/radio-buttons/), and I've actually added them to the Sketch library as a test (for now). They are under components / form controls / radio button in the Symbols menu. I'm experimenting with how we'll integrate the US system's Sketch symbols into our library, and so far I am using their naming conventions (just copy + pasting from their Sketch library to ours, plus tweaking the symbols to make them responsive and look like the coded components. One thing I was wondering was the best way to show the 8px margin that radio button options should have been them. In the older radio button symbols, the margin was built in (like you pointed out) before I started on the team. It's always kind of thrown me off, too. So I'm wondering when we release the new radio buttons if we should show a realistic example of radio button options in Sketch, Storybook or both. Thoughts?

**Comment by GnatalieH** [comment]
[Here is a link](https://www.sketch.com/s/610156b6-f281-4497-81f3-64454fc72156/a/R18WG5Z) to the new Radio button designs. I've removed the top/bottom margins from the single radio button symbols. I did that because the margin is different depending on the order of the radio button option. There is a 12px bottom margin on the radio button label/legend, and 10px top margin between radio button options. CC @andaleliz and @humancompanion-usds
